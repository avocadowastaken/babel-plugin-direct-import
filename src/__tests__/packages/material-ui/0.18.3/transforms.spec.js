const { runTransformSpec } = require('../../../../__testutils__/run-spec');

runTransformSpec(__dirname);
