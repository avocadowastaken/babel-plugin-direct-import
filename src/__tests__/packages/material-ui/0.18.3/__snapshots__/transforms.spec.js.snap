// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transform: material-ui@0.18.3 should throw with: "default-import.js" 1`] = `"unknown: babel-plugin-direct-import: material-ui does not contain module \\"MUI\\""`;

exports[`Transform: material-ui@0.18.3 should throw with: "unknown-module.js" 1`] = `"unknown: babel-plugin-direct-import: material-ui does not contain module \\"foo\\""`;

exports[`Transform: material-ui@0.18.3 should transform with: "both-modules.js" 1`] = `
import { TextField, SelectField, FlatButton } from "material-ui";
import {
  ActionAccessibility,
  ActionAccessible,
  ActionAccountBalance,
} from "material-ui/svg-icons";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import TextField from "material-ui/TextField";
import SelectField from "material-ui/SelectField";
import FlatButton from "material-ui/FlatButton";
import ActionAccessibility from "material-ui/svg-icons/action/accessibility";
import ActionAccessible from "material-ui/svg-icons/action/accessible";
import ActionAccountBalance from "material-ui/svg-icons/action/account-balance";
`;

exports[`Transform: material-ui@0.18.3 should transform with: "each-module.js" 1`] = `
import {
  AppBar,
  AutoComplete,
  Avatar,
  Badge,
  BottomNavigation,
  BottomNavigationItem,
  Card,
  CardActions,
  CardHeader,
  CardMedia,
  CardTitle,
  CardText,
  Checkbox,
  Chip,
  CircularProgress,
  DatePicker,
  Dialog,
  Divider,
  Drawer,
  DropDownMenu,
  FlatButton,
  FloatingActionButton,
  FontIcon,
  GridList,
  GridTile,
  IconButton,
  IconMenu,
  LinearProgress,
  List,
  ListItem,
  makeSelectable,
  Menu,
  MenuItem,
  MuiThemeProvider,
  Paper,
  Popover,
  RadioButton,
  RadioButtonGroup,
  RaisedButton,
  RefreshIndicator,
  SelectField,
  Slider,
  Subheader,
  SvgIcon,
  Step,
  StepButton,
  StepContent,
  StepLabel,
  Stepper,
  Snackbar,
  Tabs,
  Tab,
  Table,
  TableBody,
  TableFooter,
  TableHeader,
  TableHeaderColumn,
  TableRow,
  TableRowColumn,
  TextField,
  TimePicker,
  Toggle,
  Toolbar,
  ToolbarGroup,
  ToolbarSeparator,
  ToolbarTitle,
} from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import AppBar from "material-ui/AppBar";
import AutoComplete from "material-ui/AutoComplete";
import Avatar from "material-ui/Avatar";
import Badge from "material-ui/Badge";
import BottomNavigation from "material-ui/BottomNavigation";
import BottomNavigationItem from "material-ui/BottomNavigation/BottomNavigationItem";
import Card from "material-ui/Card";
import CardActions from "material-ui/Card/CardActions";
import CardHeader from "material-ui/Card/CardHeader";
import CardMedia from "material-ui/Card/CardMedia";
import CardTitle from "material-ui/Card/CardTitle";
import CardText from "material-ui/Card/CardText";
import Checkbox from "material-ui/Checkbox";
import Chip from "material-ui/Chip";
import CircularProgress from "material-ui/CircularProgress";
import DatePicker from "material-ui/DatePicker";
import Dialog from "material-ui/Dialog";
import Divider from "material-ui/Divider";
import Drawer from "material-ui/Drawer";
import DropDownMenu from "material-ui/DropDownMenu";
import FlatButton from "material-ui/FlatButton";
import FloatingActionButton from "material-ui/FloatingActionButton";
import FontIcon from "material-ui/FontIcon";
import GridList from "material-ui/GridList";
import GridTile from "material-ui/GridList/GridTile";
import IconButton from "material-ui/IconButton";
import IconMenu from "material-ui/IconMenu";
import LinearProgress from "material-ui/LinearProgress";
import List from "material-ui/List";
import ListItem from "material-ui/List/ListItem";
import makeSelectable from "material-ui/List/makeSelectable";
import Menu from "material-ui/Menu";
import MenuItem from "material-ui/MenuItem";
import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
import Paper from "material-ui/Paper";
import Popover from "material-ui/Popover";
import RadioButton from "material-ui/RadioButton";
import RadioButtonGroup from "material-ui/RadioButton/RadioButtonGroup";
import RaisedButton from "material-ui/RaisedButton";
import RefreshIndicator from "material-ui/RefreshIndicator";
import SelectField from "material-ui/SelectField";
import Slider from "material-ui/Slider";
import Subheader from "material-ui/Subheader";
import SvgIcon from "material-ui/SvgIcon";
import Step from "material-ui/Stepper/Step";
import StepButton from "material-ui/Stepper/StepButton";
import StepContent from "material-ui/Stepper/StepContent";
import StepLabel from "material-ui/Stepper/StepLabel";
import Stepper from "material-ui/Stepper/Stepper";
import Snackbar from "material-ui/Snackbar";
import Tabs from "material-ui/Tabs";
import Tab from "material-ui/Tabs/Tab";
import Table from "material-ui/Table";
import TableBody from "material-ui/Table/TableBody";
import TableFooter from "material-ui/Table/TableFooter";
import TableHeader from "material-ui/Table/TableHeader";
import TableHeaderColumn from "material-ui/Table/TableHeaderColumn";
import TableRow from "material-ui/Table/TableRow";
import TableRowColumn from "material-ui/Table/TableRowColumn";
import TextField from "material-ui/TextField";
import TimePicker from "material-ui/TimePicker";
import Toggle from "material-ui/Toggle";
import Toolbar from "material-ui/Toolbar";
import ToolbarGroup from "material-ui/Toolbar/ToolbarGroup";
import ToolbarSeparator from "material-ui/Toolbar/ToolbarSeparator";
import ToolbarTitle from "material-ui/Toolbar/ToolbarTitle";
`;

exports[`Transform: material-ui@0.18.3 should transform with: "svg-icons.js" 1`] = `
import {
  ActionAccessibility,
  ActionAccessible,
  ActionAccountBalance,
} from "material-ui/svg-icons";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import ActionAccessibility from "material-ui/svg-icons/action/accessibility";
import ActionAccessible from "material-ui/svg-icons/action/accessible";
import ActionAccountBalance from "material-ui/svg-icons/action/account-balance";
`;

exports[`Transform: material-ui@0.18.3 should transform with: "with-empty-specifiers.js" 1`] = `
import "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import "material-ui";
`;

exports[`Transform: material-ui@0.18.3 should transform with: "with-local-name.js" 1`] = `
import {
  TextField as MUITextField,
  SelectField as MUISelectField,
  FlatButton as MUIFlatButton,
} from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import MUITextField from "material-ui/TextField";
import MUISelectField from "material-ui/SelectField";
import MUIFlatButton from "material-ui/FlatButton";
`;

exports[`Transform: material-ui@0.18.3 should transform with: "with-namespace-specifier.js" 1`] = `
import * as all from "material-ui";
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
import * as all from "material-ui";
`;
